templates/profile.tmpl

```diff
@@ -67,6 +67,9 @@
           <tr>
             <th>Title</th>
             <th>Repository</th>
+            <th>Lang-1</th>
```

Comment:
@jcbwlkr I was thinking that one column would not sort correctly if the list was something like: `[["Go", "CSS", "HTML"], ["HTML", "Go", "CSS"]]` With one column, the first language would sort alphabetically but the second and third languages would not. Having three columns would allow for alphabetically sorting all three languages. Not sure which makes the most sense. Though, I might be making a mountain out of a mole hill lol.